omero/developers/Web/Deployment.txt

```diff
@@ -58,142 +58,7 @@ Debugging on and start the server up:
 Using FastCGI (Unix/Linux)
 --------------------------
 
-Once you have installed a FastCGI capable web server, configuration of
```

Comment:
@rleigh-dundee that section is only for developers and noone expect sysadmin to look at it. Since the begging we have 2 option to generate config:
 - nginx -> default what is below to simplify the developers work
 - nginx --sysadmin for a production deployment.
Yesterday @manics changed these options to (see https://github.com/openmicroscopy/openmicroscopy/pull/3326#issuecomment-70055266):
 - nginx - default will become sysadmin for a production deployment
 - nginx-development for developers.

If we are going to continue with that strategy we should have that here (just with new syntax) and in sysadmin. No one expect sysadmin to look at here and vice versa.